### DISCUSSION TITLE
Range/period of statistics

### DISCUSSION THREAD

**Discussion by schnillerman** [alexjustesen/speedtest-tracker#660] [Q&A] [answered]
Hi,

is there a way to go back more than one month in the statistics?

Otherwise I might raise an issue / a feature request to not only go further back in time, but also to choose a period to be displayed (e.g., the month previous to last month).

**Comment by alexdelprete** [comment, marked as answer]
For long-term data queries and graphs, I would advice to use the integrated influxdb exporter with Grafana. I created a ready-to-use dashboard for Grafana that you can find [here](https://grafana.com/grafana/dashboards/17460-speedtest-tracker-v2/).

![image](https://github.com/alexjustesen/speedtest-tracker/assets/7027842/e2c8421c-49ae-4119-a67f-306239ed97ba)
